### DISCUSSION TITLE
Simulation stage results

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas24_competition#52] [Announcements]
## Here there are, gj everyone
| Rank | Team                    | Eval1 | Eval2 | Eval3 | Eval4 | Eval5 | Eval6 | Bo4 AVG |
|------|-------------------------|-------|-------|-------|-------|-------|-------|---------|
| 1    | ITUAV                   | 0     | 89,76 | 97,33 | 98,73 | 0     | 94,65 | 95,12   |
| 2    | UAS-DTU                 | 0     | 0     | 97,91 | 92,21 | 96,62 | 92,45 | 94,80   |
| 3    | ASL                     | 0     | 84,89 | 94,65 | 85,11 | 97,18 | 91,71 | 92,16   |
| 4    | AiRoLab                 | 0     | 0     | 94,25 | 93,45 | 77,28 | 86,7  | 87,92   |
| 5    | AVANT-UFMG              | 0     | 0     | 86,44 | 83,68 | 88,58 | 84,5  | 85,80   |
| 6    | AVADERAGH               | 0     | 32,03 | 90,49 | 89,9  | 72,13 | 86,8  | 84,83   |
| 7    | UMIC                    | 0     | 57,70 | 73,86 | 93,71 | 85,68 | 38,16 | 77,74   |
| 8    | SDCNLab                 | 0     | 72,06 | 79,96 | 72,26 | 69,04 | 70,25 | 73,64   |
| 9    | CVAR                    | 63,05 | 64,43 | 70,58 | 79,7  | 66,84 | 66,46 | 70,40   |
| 10   | FeikAir                 | 0     | 0     | 75,54 | 68,33 | 71,05 | 65,82 | 70,19   |
| 11   | high_figh_fly           | 43,57 | 24,62 | 50,28 | 60,35 | 68,48 | 51,78 | 57,72   |
| 12   | LCIS                    | 30,06 | 61,18 | 65,91 | 20,13 | 28,43 | 64,47 | 55,41   |
| 13   | CapyBots                | 0     | 0     | 25,63 | 22,24 | 38,38 | 30,39 | 29,16   |
| 14   | SkoltechHawks           | 0     | 0     | 28,08 | 28,09 | 24,32 | 23,29 | 25,95   |
| 15   | percepDrone             | 0     | 0     | 17,66 | 17,65 | 18,43 | 19,56 | 18,33   |
| 16   | TeamHawkeye             | 0     | 8,55  | 32,13 | 25,93 | 0     | 0     | 16,65   |
| 17   | BDPFly                  | 0     | 0     | 53,94 | 0     | 0     | 0     | 13,49   |
| 18   | AeroMIT                 | 0     | 0     | 0     | 41,83 | 0     | 0     | 10,46   |
| 19   | AerialRoboticsIITK      | 0     | 0     | 0     | 0     | 0     | 0     | 0       |
| 19   | AerialRoboticsKharagpur | 0     | 0     | 0     | 0     | 0     | 0     | 0       |
| 19   | DMA                     | 0     | 0     | 0     | 0     | 0     | 0     | 0       |
| 19   | UniboUAV                | 0     | 0     | 0     | 0     | 0     | 0     | 0       |
| -    | Sandrone                | 0     | 0     | 75,93 | 89,90 | 86,00 | 84,10 | 84,23   |

## Run logs, scenarios and stuff
Will be posted and sent over the weekend. We will also respond to emails sent over the last week as soon as 

## Finals news
Unfortunately, as it stands now, we will not be able to set the arena up at the conference. Details on the finals will be hopefully set over the next week, and we will update you all as soon as possible.

**Comment by cmakelabs** [comment]
Execuse me, may I know why Sandrone has no rank?

**Reply by fpetric** [reply]
They were a bit late to submit so we wanted to acknowledge their good results, but could not place them above the teams that submitted on time.

**Comment by joaovmamute** [comment]
Hello, I would like to know if the teams will receive feedback of the results

**Reply by fpetric** [reply]
It should have been sent to team leaders. Let me know via email if your team did not receive it.

**Comment by Zeezus11** [comment]
Waiting for the details of the final round but may I know whether the finals will be offline or online. 

Thanks

**Comment by Zeezus11** [comment]
Hi, it's been a week now, please share the details of finals.
Thanks

**Comment by chirAAG-sehgal** [comment]
Any updates regarding the final competition @fpetric ?
